.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
 env:
   global:
     - SYMFONY_PHPUNIT_DIR=./bin/.phpunit
-    - SYMFONY_DEPRECATIONS_HELPER=0
+    - SYMFONY_DEPRECATIONS_HELPER=9
```

Comment:
Hi, @javiereguiluz  What is 9 ? :thinking:

Reply:
They are the Doctrine deprecations that I couldn't remove and prevented tests from being green. See https://travis-ci.org/symfony/demo/jobs/627625205#L358